docs/api-versions.md

```diff
@@ -0,0 +1,17 @@
+# API versions
+
+## Supported API Versions
+
+The Performance Addon Operator supports *v1* and *v1alpha1* [Performance Profile](docs/performance_profile.md) versions.
+There are no differences between the *v1* and *v1alpha1* versions, the upgrade has been made in order to
+mark the Performance Profile API as stable.
+
+## Upgrade from *v1alpha1* to *v1*.
+When upgrading from an older Performance Addon Operator version supporting only *v1alpha1* API to a newer one supporting
+also the *v1* API, the existing *v1alpha1* Performance Profiles will be converted on-the-fly using a "None" Conversion
+strategy and served to the Performance Addon Operator as *v1*.
+
+## Q&A
+What happens in practice if I install a v1-enabled PAO on a cluster? What should I expect?
+
+PAO will expect v1 Performance Profiles and query only them. Existing v1alpha1 profiles will be served as v1 and PAO works with them as usual.
```

Comment:
so this means that existing v1alpha1 profiles will be stored in the cluster as v1 objects, right?
IOW, even if we submit a v1alpha1 profile, we will always get back a v1 profile, right?
Same: in etcd there always is v1 objects, isn't it?
This is what should be clarified in this doc: we can obviously learn the same from the code and/or from the functests, but the point of the docs is allowing to avoid that :)

Reply:
Submitting a new v1alpha1 profile after the PerformanceProfile has been updated to v1 will be stored as a v1 profile.
You can retrieve it as both v1alpha1 and v1 profile.
I am not sure if the existing v1alpha1 profiles are converted into v1 as part of the upgrade and stored in etcd as v1 or remain v1alpha1 and get converted on the fly. I am not sure it matters for us.

I am not sure this resolution is part of the PAO doc though

Reply:
ok, let's just add this to the A above and that's good enough.